src/app/flowtype/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client"
+
+import React from 'react';
+import TileSelector from './TileSelector';
+
+const FlowType: React.FC = () => {
```

Comment:
We don't need a page or client page for this component, only the UI component. The parent tasks with the page and client pages are already being worked on by Shivam here: https://issues.redhat.com/browse/SCO-74